src/components/Staking/index.tsx

```diff
@@ -29,8 +28,8 @@ export default function StakingBox(props: Props) {
       <Container
         mt={20}
         maxW={"7xl"}
-        columns={{ base: 1, md: 2 }}
-        spacing={{ base: 10, lg: 32 }}
+        // columns={{ base: 1, md: 2 } as any}
+        // spacing={{ base: 10, lg: 32 }}
       >
```

Comment:
Is this still in use? If not, please remove it.

Reply:
I resolved it